PowerPointLabs/PowerPointLabs.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15

```

Comment:
Any idea why the solution file is changed? Is this required?

Reply:
nope, it's not required